docs/designs/ps_design.md

```diff
@@ -1,377 +1,94 @@
 # Parameter Server Design
+This document describes the design of a distributed parameter server (PS) to support training scalability and PS fault tolerance in ElasticDL.
```

Comment:
The appearance of "ElasticDL" is abrupt. We could change the title to be "The Parameter Server of ElasticDL", or talk about ElasticDL by inserting a sentence before this one.

Reply:
The concept of fault-tolerance appears abruptly too. Reader would be surprised -- why do we need fault-tolerance?

Reply:
fault tolerance => fault-tolerance